i18n-helpers/src/normalize.rs

```diff
@@ -247,10 +248,19 @@ pub fn normalize(catalog: Catalog) -> anyhow::Result<Catalog> {
                 message.source_mut().push('\n');
                 message.source_mut().push_str(new_message.source());
             }
-            None => new_catalog.append_or_update(new_message),
+            None => new_catalog.append_or_update(new_message)
         }
     }
 
+    // Increment POT-Creation-Date in metadata by 1 minute to indicate a change in the file.
+    if let Ok(pot_creation_date) = dateparser::parse(&new_catalog.metadata.pot_creation_date) {
+        new_catalog.metadata.pot_creation_date = pot_creation_date
+            .checked_add_signed(Duration::minutes(1))
+            .ok_or(anyhow::Error::msg("could not advance POT-Creation-Date"))?
```

Comment:
Could you instead leave the date unchanged if it cannot be parsed?

Reply:
Done!